libs/pika/command_line_handling/src/parse_command_line.cpp

```diff
@@ -373,6 +373,11 @@ namespace pika::detail {
                   "values. Do not use with --pika:pu-step, --pika:pu-offset, or "
                   "--pika:affinity options. Implies --pika:numa-sensitive=1"
                   "(--pika:bind=none disables defining thread affinities).")
+                ("pika:override-process-mask", value<std::string>(),
+                  "a process mask in hexadecimal form to override a mask on the "
+                  "executable. Has no effect if --pika:ignore-process-mask is used. "
+                  "This option does not set the process mask for the main thread. "
+                  "The mask only affects threads spawned by the pika runtime.")
```

Comment:
Should this simply be called `--pika:process-mask`? In cases where there is no special mask set (like on a desktop or laptop) it's not really overriding anything (just "overriding" the full machine mask which is always there if nothing else is set). Another option is `--pika:set-process-mask` but the `set` feels superfluous to me. Most command line options set something...

Reply:
I think it might be a better name, for the reason you suggest. It also makes a bit more sense that setting `--pika:process-mask` is ignored if `--pika:ignore-process-mask`, while the `--pika:override-process-mask` name is a bit more confusing in such context IMHO.